kinetic_maintainer_update/cmake/functions.cmake

```diff
@@ -0,0 +1,134 @@
+# Copyright 2016 The Cartographer Authors
```

Comment:
can you please rename these folders to just be the distro name as they now contain more changes than maintainer name updates?